waku/node/peer_manager/peer_manager.nim

```diff
@@ -959,6 +964,7 @@ proc new*(
     switch: Switch,
     wakuMetadata: WakuMetadata = nil,
     maxRelayPeers: Option[int] = none(int),
+    maxServicePeers: Option[int] = none(int),
```

Comment:
Is the peer manager already initialized with `maxServicePeers` somehow?

Reply:
Yeah, it's not passed anywhere. Instead of making it a local variable, I thought it would be better to incorporate it into the peer_manager. This would showcase the functionality of peer_manager and also make it easier to add some unittests.